### ISSUE TITLE
Overclocks showing "Forged" but doesn't show up in game

### ISSUE THREAD

**Issue by SunnivaClair** [AnthonyMichaelTDM/DRG-Save-Editor#86]
I don't think this a bug with the save editor itself, but more that I have a corrupted save file and don't know how to repair it. If this isn't the right place to post this please remove this issue, this just seems like the most active save editor so am wondering if y'all have any insight into my issue.

# Issue

Before the recent Season 5 update I used a save editor (I think it was one of the online editors) to unlock all the overclocks I hadn't gotten yet since I had taken a couple year break from the last time I was in the game. I didn't really verify that it actually unlocked all the overclocks because I wasn't familiar with any of the new secondaries. I ended up making two major mistakes though: I lost the backup of the save before I edited it and I ended up putting a lot of hours on the new save. I now have unlocked all the new Season 5 overclocks and am not getting any more weapon overclocks from missions/machine events/deep dives, but after checking the wiki am missing a lot of overclocks.

Uploading the save to the save editor or to DRG Completionist shows that I have every overclock unlocked, but when I go into the weapon terminal I don't see quite a few, as listed below.

### Engineer

- Efficiency Tweaks
- Automated Beam Controller
- Volatile Impact Reactor
- Armor Breaking Module
- Executioner

### Gunner

- Re-atomizer
- Backfeeding Module
- The Mole
- Hellfire
- Triple-Tech Chambers

### Scout

- Cryo Bolt
- Fire Bolt
- Bodkin Points

# Troubleshooting I've Taken

- Using [this](https://github.com/phillihp-rockade/drg-editor) editor I've tried to set the overclocks to unforged or even unaquired but whenever I load the save in game it resets all my progress to zero.

- I've loaded it to this save editor but don't see any way to set an overclock to unforged. 

I've attached my save file if anyone wants to take a look at it. I really don't want to reset my save game I've been playing since 2019, but also am missing a ton of overclocks that I would like to use.

**Comment by SunnivaClair** [comment]
Sorry, forgot to attach save. Also changed it to a txt file since I can't upload .sav ? Lmk if that messes anything up

[76561198268846052_Player.txt](https://github.com/user-attachments/files/16216118/76561198268846052_Player.txt)

**Comment by AnthonyMichaelTDM** [comment]
this seems similar to #80, but still worth looking at. Thanks for the detailed report

**Comment by AnthonyMichaelTDM** [comment]
before I dig into this, have you looked at https://github.com/MrMarble/drg-editor/issues/33 ?

**Comment by AnthonyMichaelTDM** [comment]
oh wait, nvm, that comment is saying to use this tool :skull:

**Comment by SunnivaClair** [comment]
Yeah, looks like the same or at least similar problem to both those issues. I'll try the process the comment suggests tomorrow and report if it works or not!

**Comment by AnthonyMichaelTDM** [comment]
sure thing

**Comment by AnthonyMichaelTDM** [comment]
also, see if this file works, I tried removing the OC's with that editor you linked earlier then re-adding them with this editor
[76561198268846052_Player maybe fixed.txt](https://github.com/user-attachments/files/16216398/76561198268846052_Player.maybe.fixed.txt)

**Comment by SunnivaClair** [comment]
> also, see if this file works, I tried removing the OC's with that editor you linked earlier then re-adding them with this editor [76561198268846052_Player maybe fixed.txt](https://github.com/user-attachments/files/16216398/76561198268846052_Player.maybe.fixed.txt)

Tried this file and when I launch the game it resets my save to blank

Also tried doing it myself from the instructions from https://github.com/MrMarble/drg-editor/issues/33#issuecomment-1368107349 and got the same result of the game resetting my save

I'm really curious about why there (I assume) to be two places that determine if you have a overclock unlocked that can desyncronize :(

Let me know if there is anything more I can do to help!

**Comment by AnthonyMichaelTDM** [comment]
Yeah, it's kinda funky like that.
This is the main reason why this editor does not provide a way to forge OCs directly.

**Comment by SunnivaClair** [comment]
Damn, that rough to hear :( I'll make sure to use this editor in the future since it seems to be a lot more stable

Good news though! I found the old save file before I edited it originally! I've lost roughly 40 hours of progress, but that is a lot better than 400 :sob: 

I'll attach it in case anybody wants to take a look at it to see if they can figure out if there is a future solution to resolve these corrupted files?
[76561198268846052_Player.txt](https://github.com/user-attachments/files/16229291/76561198268846052_Player.txt)

**Comment by AnthonyMichaelTDM** [comment]
Oh that's great news!

Actually, if you have a hex editor that can do diffing (like HxD or imhex), you could compare the old save to the corrupted save, which might provide some useful insights
